angular-client/src/app/app-nav-bar/app-nav-bar.component.html

```diff
@@ -0,0 +1,43 @@
+@if (isMobile) {
+  <p-sidebar [(visible)]="sidebarVisible">
+    <ng-template pTemplate="header">
+      <typography style="width: 100%; text-align: center" variant="large-header" content="Argos" />
+    </ng-template>
+    <ng-template pTemplate="content">
+      <sidebar-chip icon="home" value="Home" [active]="selectedRoute === '/landing'" (click)="navigateTo('/landing')">
```

Comment:
turn these into an array of objects in the ts file and then iterate over them putting them here